test/geometry/adjoints.jl

```diff
@@ -53,25 +53,51 @@ end
     end # TODO: Remove the @test_broken after tests work.
 end
 
-@testset "SVector{N, T} gradient" begin
+@testset "SVector{N, T} and SMatrix{N, N, T, L} gradient" begin
```

Comment:
Nobody will be interested in the types, just "StaticArrays" would be okay since this testset is already a subset of `Adjoints` group.